tests/pytorch/mock_sparse2/test_sparse_matrix.py

```diff
@@ -10,15 +10,14 @@
 if not sys.platform.startswith("linux"):
     pytest.skip("skipping tests on win", allow_module_level=True)
 
+
 @pytest.mark.parametrize("dense_dim", [None, 4])
-@pytest.mark.parametrize("row", [[0, 0, 1, 2], (0, 1, 2, 4)])
+@pytest.mark.parametrize("row", [(0, 0, 1, 2), (0, 1, 2, 4)])
 @pytest.mark.parametrize("col", [(0, 1, 2, 2), (1, 3, 3, 4)])
 @pytest.mark.parametrize("shape", [None, (3, 5), (5, 3)])
 def test_create_from_coo(dense_dim, row, col, shape):
     # Skip invalid matrices
-    if shape is not None and (
-        max(row) >= shape[0] or max(col) >= shape[1]
-    ):
+    if shape is not None and (max(row) >= shape[0] or max(col) >= shape[1]):
```

Comment:
I don't like this. Can we adjust the test cases to rule out invalid combinations?